red_lower : 150
red_upper : 255
green_lower : 140
green_upper : 210
blue_lower : 0
blue_upper : 85
